pkg/web/templates/accounts.html

```diff
@@ -47,7 +47,8 @@ <h1 class="font-bold text-xl">Add New Customer Account</h1>
               <input type="text" id="crypto_address_0" name="crypto_address_0" class="input-style" />
             </div>
             <div>
-              {{ template "network_select" . }}
+              <label for="networks" class="label-style">Network</label>
+              <select id="networks" name="network_0"></select>
```

Comment:
When the select was added for network, the number was removed from the name and that's why network wasn't included in the request. There's a separate story to handle replacing inputs with selects for additional wallets in the add account modal.